android/src/com/frostwire/android/gui/adapters/menu/FileListAdapter.java

```diff
@@ -90,7 +89,7 @@ public byte getFileType() {
 
     @Override
     protected final void populateView(View view, FileDescriptorItem item) {
-        if (getViewItemId() == R.layout.view_browse_thumbnail_peer_list_item) {
+        if (fileType == Constants.FILE_TYPE_PICTURES || fileType == Constants.FILE_TYPE_VIDEOS || fileType == Constants.FILE_TYPE_APPLICATIONS || fileType == Constants.FILE_TYPE_AUDIO) {
```

Comment:
I also wonder if that method should be the other way around logically (for less code)
```
# pseudo code
return fileType not in (torrents, docs)
```